ruby/rails/security/brakeman/check-redirect-to.yaml

```diff
@@ -0,0 +1,71 @@
+rules:
+- id: check-redirect-to
+  mode: taint
+  pattern-sources:
+    - patterns:
+        - pattern-either:
+            - pattern: params
+            - pattern: cookies
+            - pattern: request.env
+            - pattern: url_for(params[...],...,:only_path => false,...)
+  pattern-sanitizers:
+    - patterns:
+        - pattern-either:
+            - patterns:
+              - pattern: |
+                  $F(...)
+              - metavariable-pattern:
+                  metavariable: $F
+                  patterns:
+                      - pattern-not-regex: (params|url_for|cookies|request.env|permit|redirect_to)
```

Comment:
This is a galaxy-brain level hack :O

Reply:
it's so ugly but it does what I need. I am shamed by how long it took me to come up with it.